plan/config.go

```diff
@@ -104,10 +114,54 @@ func parseBehavior(d string) Behavior {
 		ClientAxis: clientAxis,
 		ParamsAxes: values,
 	}
-
 	return behavior
 }
 
+func validateAndApplyFilters(filterMap map[string][]Filter, behaviors Behaviors) (Behaviors, error) {
+	for i := range behaviors {
```

Comment:
Yep, I'd a good learning here. If I do that, the object of behavior that is updated is not reflected in calling function.